doc/modules/cassandra/pages/managing/operating/compression.adoc

```diff
@@ -298,12 +298,14 @@ next access.
 
 === Training Configuration
 
-* `compression_dictionary_training_max_dictionary_size` (default: `65536`):
+* `training_max_dictionary_size` (default: `65536`):
 Maximum size of trained dictionaries in bytes. Larger dictionaries can
-capture more patterns but increase memory overhead.
-* `compression_dictionary_training_max_total_sample_size` (default:
+capture more patterns but increase memory overhead. This is a parameter
+of `ZstdDictionaryCompressor` of a table, in `compression` section.
+* `training_max_total_sample_size` (default:
 `10485760`): Maximum total size of sample data to collect for training,
-approximately 10MB.
+approximately 10MB. This is a parameter of `ZstdDictionaryCompressor`
+of a table, in `compression` section.
```

Comment:
Those are to document the configs in cassandra.yaml. Since they are removed, maybe remove from here as well, or clarify that they are referring to the parameters in the compression attribute in table schema.